jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull.yaml

```diff
@@ -209,11 +210,7 @@
 
               # GKE variables
               export CLUSTER_NAME=${{E2E_NAME}}
-              export ZONE="us-central1-f"
               export KUBE_GKE_NETWORK=${{E2E_NAME}}
-              export CLOUDSDK_BUCKET="gs://cloud-sdk-testing/ci/staging"
```

Comment:
It means we're testing GKE cluster creation as well as tests. It'll be a few days before I can turn it on in test.